diesel/src/query_builder/for_update_clause.rs

```diff
@@ -12,4 +12,15 @@ impl<DB: Backend> QueryFragment<DB> for NoForUpdateClause {
 }
 
 #[derive(Debug, Clone, Copy, QueryId)]
-pub struct ForUpdateClause;
+pub struct ForUpdateClause<Modifier> {
+    pub(crate) modifier: Modifier,
+}
+
+#[derive(Debug, Clone, Copy, QueryId)]
+pub struct NoModifier;
+
+#[derive(Debug, Clone, Copy, QueryId)]
+pub struct SkipLockedModifier;
```

Comment:
Do we need `Modifier` at the end of every type? I think just `SkipLocked` is fine.

Reply:
OK